setup.py

```diff
@@ -17,28 +17,42 @@
 
 scripts = ['tx']
 
-install_requires = []
+install_requires = ['pyinstaller']
 extra_args = {}
 import platform
 if platform.system() == 'Windows':
```

Comment:
Just want to make it clear. We should just get rid of this `if platform.system() == 'Windows'` and *all content under* it. We just need the `tx.spec` working fine when issuing. 

I believe generating the `tx.exe` in `pip install` is a bad idea because we won't be able to uninstall it on `pip uninstall.